demo/lightning/image-segmentation/model.py

```diff
@@ -56,7 +56,8 @@ def configure_optimizers(self):
     def training_step(self, train_batch, batch_idx):
         images, labels = train_batch
         if self.benchmark:
-            return None
+            # Return a scalar of value 1 to speed up training_step.
```

Comment:
nit: This comment doesn't really explain why this speeds it up compared to returning None. Is there any way to add that, either to the comment or to the PR description?

Reply:
Updated the comment just now :)

Reply:
Also updated PR description.